src/server/middleware/helmet.ts

```diff
@@ -61,10 +61,12 @@ const defaultSrc = [
   'https://api-js.mixpanel.com',
   'checkoutshopper-live.adyen.com',
   'checkoutshopper-test.adyen.com',
-  'aff.addreax.com',
+  'https://aff.addreax.com',
   'js.go2sdk.com',
   'secure.adnxs.com',
   'snap.licdn.com',
+  'https://bat.bing.com/bat.js',
```

Comment:
When trying out this locally, this still got blocked, it only wants the domain :)
```suggestion
  'https://bat.bing.com',
```

Reply:
Ah, of course! 🧠